impeller/aiks/aiks_blur_unittests.cc

```diff
@@ -76,8 +76,9 @@ TEST_P(AiksTest, CanRenderForegroundAdvancedBlendWithMaskBlur) {
 
 TEST_P(AiksTest, CanRenderBackdropBlurInteractive) {
   auto callback = [&](AiksContext& renderer) -> std::optional<Picture> {
-    auto [a, b] = IMPELLER_PLAYGROUND_LINE(Point(50, 50), Point(300, 200), 30,
-                                           Color::White(), Color::White());
+    static PlaygroundPoint point_a(Point(50, 50), 30, Color::White());
+    static PlaygroundPoint point_b(Point(300, 200), 30, Color::White());
+    auto [a, b] = DrawPlaygroundLine(point_a, point_b);
```

Comment:
If the intention behind this is change is make tests where these are used golden-able, would just changing the macros to forward defaults when `IMPELLER_GOLDEN_TESTS` is enabled work?

Reply:
The intention is to get this code compliant with the c++ style guide: https://google.github.io/styleguide/cppguide.html#Preprocessor_Macros

My next PR will get these running in golden image tests.  This is just cleanup.  That's an interesting idea with IMPELLER_GOLDEN_TESTS.  I will probably avoid latching more decisions on preprocessor macros and instead rely on AiksPlayground vs GoldenPlayground.

Reply:
Oh cool, I was able to just gate the logic on the presence of an ImGui context.  I'll include you in the follow up PR.

Reply:
Here it is: https://github.com/flutter/engine/pull/50606

Reply:
Makes sense... At some point we could change this to use ImGui's actual identity/storage system so that they can remain as one function without the statics. Added an issue [here](https://github.com/flutter/flutter/issues/143383).